src/lib/transaction-manager/index.ts

```diff
@@ -140,6 +140,26 @@ export default class TransactionManager {
     return await this.submitTransaction(transactions);
   }
 
+  public async setSchema(schema: Schema) {
+
+    const now = new Date().getTime();
+    const transactions = [{
+      id: uuid(),
+      operations: [
+        {
+          id: this.keys?.collectionId,
+          table: 'collection',
+          path: [],
+          command: 'update',
+          args: {
+            schema
+          },
+        }
+      ]
```

Comment:
i wanted to include a 'last_edited_time' event, which is what the website does, but doesn't seem to be required. I kept getting a "PostgresNullConstraintError" unless I used the pageId, which I don't have access to from the table instance. Any suggestions?